otx/cli/tools/deploy.py

```diff
@@ -78,9 +91,11 @@ def main():
 
     deployed_model = ModelEntity(None, environment.get_model_configuration())
 
+    if "save_model_to" not in args or not args.save_model_to:
+        args.save_model_to = str(config_manager.workspace_root / "model-deployed")
     os.makedirs(args.save_model_to, exist_ok=True)
     task.deploy(deployed_model)
-    with open(os.path.join(args.save_model_to, "openvino.zip"), "wb") as write_file:
+    with open(str(Path(args.save_model_to) / "openvino.zip"), "wb") as write_file:
```

Comment:
```suggestion
    with open(Path(args.save_model_to) / "openvino.zip", "wb") as write_file:
```
open can get Path.

Reply:
reflected this. https://github.com/openvinotoolkit/training_extensions/pull/1641/commits/3fcf067f3ae5f70826b0df66bdb702db0a04c358